packages/ses/src/lockdown-shim.js

```diff
@@ -24,7 +24,11 @@ import tameMathObject from './tame-math-object.js';
 import tameRegExpConstructor from './tame-regexp-constructor.js';
 import enablePropertyOverrides from './enable-property-overrides.js';
 import tameLocaleMethods from './tame-locale-methods.js';
-import { initGlobalObject } from './global-object.js';
+import {
+  initGlobalObjectConstants,
+  initGlobalObjectProperties,
```

Comment:
If the distinction is constant versus variable properties, these names could paint the distinction better. Perhaps `initGlobalObjectConstantProperties` vs `initGlobalObjectVariableProperties`.

If the distinction is shared intrinsics, maybe we can fit shared intrinsics in the name.

Reply:
I suppose it'd be the former since both setup different shared intrinsics. I honestly didn't think much of the naming, I'll run with yours.